src/settings/general-settings.tsx

```diff
@@ -34,6 +34,23 @@ const GeneralSettings: FC = () => {
 			}
 		}));
 	}, []);
+	const removeMod = useCallback((type: 'props' | 'prefs', key) => {
+		setMods((prevState) => {
+			const prevType = prevState[type];
+			if (prevType && prevType[key]) {
+				const nextState = { ...prevState, [type]: { ...prevState[type] } };
+				const nextType = nextState[type];
+				if (nextType && nextType[key]) {
```

Comment:
this can create bugs: if `nextType[key]` is indeed present but it is set to 0, false or any other value which is translated to false by js, the next line won't be executed and so the value won't be removed from the object, resulting in weird unwanted behaviours